removeDuplicatesFromSortedListII.py

```diff
@@ -0,0 +1,29 @@
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.next = None
+
+class Solution:
+    def deleteDuplicates(self, head):
+        """
+        :type head: ListNode
+        :rtype: ListNode
+        """
+        dummy = ListNode(2 ** 31 + 1)
```

Comment:
You don't need `2 ** 31 + 1`.
Can just do `head.val - 1`.
But make sure you do `if not head or not head.next` first.